compliance_checker/ioos.py

```diff
@@ -25,9 +28,6 @@ def _has_attr(cls, ds, attr, concept_name, priority=BaseCheck.HIGH):
 
         return Result(priority, val, concept_name, msgs)
 
-
-class IOOSNCCheck(BaseNCCheck, IOOSBaseCheck):
-
     @classmethod
     def _has_var_attr(cls, dataset, vname, attr, concept_name, priority=BaseCheck.HIGH):
```

Comment:
Just a minor comment here: `_has_var_attr` sounds very similar to the Python builtin `hasattr`.  We may want to change this down the road.